### ISSUE TITLE
Find out the limits of the $RANDOM special variable

### ISSUE THREAD

**Issue by lcabrini** [Prince-linux/prince-linux-learning#40]
Create a wiki page, called Random Limits, and describe how to find out the limits of the $RANDOM variable, that is the lowest and the highest number that can be returned by $RANDOM. Explain your solution.

Then write a script, called random-limits.sh where you put your idea into practice. The program should produce output as the following:

```
Minimum: 1
Maximum: 1500
```

That is, if the limits of $RANDOM were 1 to 1500 (which they are not).

**Comment by lcabrini** [comment]
How are you getting along with this issue? Just comment here if you get stuck.

**Comment by Prince-linux** [comment]
hmm 
I run this command at a website I found while researching and bash gave me an error but still run. Don't know why
 this is the command  
(( ( RANDOM % 10 )  + 1 ))

**Comment by lcabrini** [comment]
to make use of that you would need to do something like:

```
r=$(( ( RANDOM % 10 ) + 1 ))
echo $r
```

which is fine for what it does, but it is not going to give you the limits. It is going to generate random numbers 1-10, since the remainder of division by 10 can only be a number in the range 0-9, and adding 1 to that you get a number in the range 1-10 instead. This is useful in some cases, and you will use this fact in another task.

But in this case you want to test the limits of $RANDOM itself. That is, you want to know the lowest and the highest values that it can produce. Here is a breakdown of the sequence.
1. Set up variables $min and $max
2. Loop a large number of times
3. In the loop generate a random number and store it in a variable, such as $r
4. compare the new $r to $max, if it is greater than $max set $max to $r
5. compare the new $r to $min, if is is less than $min, set $min to $r
6. Once the loop is done, print out the values of $min and $max

If the number of loop iterations is large enough, you should get the actual limits. Notice that you will need to initialize $max to a very small number and $min to a very large number. Why is that so? Explain by adding comments in the script above the lines where you initialize the variables $min and $max.

If the program is working properly, you should be able to run it multiple times and always get the same result. Since we are dealing with random numbers, there is always a chance that it doesn't work out every time, but statistically, if the number of loop iterations is large enough, you should get the correct result most of the time.
